src/engine/openspaceengine_lua.inl

```diff
@@ -235,4 +235,24 @@ namespace {
     openspace::setCameraFromProfile(*openspace::global::profile);
 }
 
+/**
+ * Returns the whole configuration object as a Dictionary
+ */
+[[codegen::luawrap]] ghoul::Dictionary configuration() {
+    ghoul::Dictionary res;
```

Comment:
Can move the `ghoul::Dictionary res;` declaration down to line 246. No need to have the value while accessing the configuration